src/task/cleanup.php

```diff
@@ -0,0 +1,38 @@
+<?php declare(strict_types=1);
+
+namespace Friemt\Deployer\Tasks;
+
+use function Deployer\get;
+use function Deployer\run;
+use function Deployer\set;
+use function Deployer\task;
+use function Deployer\test;
+use function Deployer\within;
+use function Deployer\writeln;
+
+set('cleanup_paths', ['var/cache']);
+
+task('cleanup:paths', function (): void {
+    $currentRelease = basename(get('release_or_current_path'));
+    $releases = get('releases_list');
+    $sudo = get('cleanup_use_sudo') ? 'sudo' : '';
```

Comment:
Also add the variable to the readme file.

Reply:
It's an already existing variable [provided by deployer](https://deployer.org/docs/7.x/recipe/deploy/cleanup#cleanup_use_sudo)